docs/guides/cloning-git-repositories.md

```diff
@@ -0,0 +1,66 @@
+# Cloning Git Repositories
+
+<div style="padding: 0px; margin: 0px;">
+  <span style="vertical-align:middle;">Author: </span>
+  <a href="https://github.com/Emyrk" style="text-decoration: none; color: inherit; margin-bottom: 0px;">
+    <span style="vertical-align:middle;">Bruno Quaresma</span>
+    <img src="https://avatars.githubusercontent.com/u/3165839?v=4" width="24px" height="24px" style="vertical-align:middle; margin: 0px;"/>
+  </a>
+</div>
+August 01, 2024
+
+---
+
+When starting to work on a project, engineers usually need to clone a Git
+repository. Even though this is often a quick step, Coder can automate it to
+make the flow more seamless, creating workspaces ready for development and
```

Comment:
I'd make a small rewrite from

> Even though this is often a quick step, Coder can automate it to make the flow more seamless, creating workspaces ready for development and reducing onboarding time.


To

> Even though this is often a quick step, it can be automated using the [Coder Registry](https://registry.coder.com/modules) to make a seamless Git-first workflow.